cert/aziot-certd/src/error.rs

```diff
@@ -48,26 +48,28 @@ impl std::error::Error for Error {
 #[derive(Debug)]
 pub enum InternalError {
     CreateCert(Box<dyn std::error::Error + Send + Sync>),
-    DeleteFile(std::io::Error),
     GetPath(Box<dyn std::error::Error + Send + Sync>),
     InvalidProxyUri(Box<dyn std::error::Error + Send + Sync>),
     LoadKeyOpensslEngine(openssl2::Error),
+    DeleteFile(std::io::Error),
```

Comment:
Variants should be in alphabetical order.

Reply:
[Updated](https://github.com/Azure/iot-identity-service/blob/6dc85fd27362b0013551bc39f633711bad41c84f/cert/aziot-certd/src/error.rs#L49-L90)